components/supervisor-api/status.proto

```diff
@@ -119,3 +126,15 @@ message PortsStatus {
 
     uint32 global_port = 2;
 }
+
+message TasksStatusRequest {    
+}
+message TasksStatusResponse {
+    repeated TasksStatus tasks = 1;
+}
+message TasksStatus {
+    string alias = 1;
+    string name = 2;
+    string openIn = 3;
```

Comment:
in gRPC fields use `_` instead of camel case. The `protoc` then adopt the convention most suited for the target language (i.e. `open_in` becomes `OpenIn` in Go and `openIn` in TS).